### ISSUE TITLE
Redundant links on Vet Center home page

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#24749]
## Issue Description
This issue was found during testing in preparation for staging review. 

**Defect**
Adjacent links (within the photo and map) go to the same URL.
Why It Matters: this results in additional navigation and repetition for keyboard and screen reader users

![image](https://user-images.githubusercontent.com/55411834/118335254-7ab5e080-b4cc-11eb-8e34-e4322dc92a45.png)

**Code**

```
<a id="generated-mapbox-image-link" href="https://maps.google.com?saddr=Current+Location&daddr=3500 Ludington Street, Suite 110, Escanaba, MI 49829" target="_blank" rel="noopener noreferrer">
<div class="va-c-position--relative vads-u-display--inline-block">
<span class="vads-u-margin-right--1p5 vads-u-margin-top--1p5 vads-u-text-decoration--none vads-u-display--flex vads-u-align-items--center expand-image-button va-c-position--absolute va-c-position-top-right-corner vads-u-justify-content--center">
<i class="fas fa-expand-arrows-alt"></i>
</span>
```

---
## Tasks
- [ ] First step: remove redundant link
- [ ] Add listener to track clicks on the image
- [ ] Map link should be maintained from the map thumbnail 

## Acceptance Criteria
- [ ] Adjacent links will be discernable
- [ ] The photo is not clickable
- [ ] Clicks on the photo are tracked
- [ ] Veteran can view the map in a new tab by clicking on the map

**Comment by mmiddaugh** [comment]
Idea for future state: Photo should open into a larger image. Veteran can view a larger image by clicking on the facility photo. We'll use analytics to determine if we need to return to this feature. The ability to increase the photo size may depend on the resolution of the photo and CMS handling.

**Comment by mmiddaugh** [comment]
Closing this issue as it duplicates 508-defect-2 [SCREENREADER, COGNITION]: Images of the facilities should not be links
#25023
